README.md

```diff
@@ -1,33 +1,80 @@
+**Delivery mindset**
+
+We believe in being pragmatic and moving fast, spending our energy on the right things, shipping code and then rapidly iterating on it. 
+
+_Behaviours_:
+- We simplify our work, focusing on getting it to the user as early as possible.
+- We practice [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery)
+- We are trying to [make the change easy, before the easy change](https://twitter.com/KentBeck/status/250733358307500032).   
+- We aim to solve issues rapidly prioritising them based on impact they may have.
+
+**Inclusiveness & Respect** 
 
 We believe that our culture is important, and we all contribute to it regardless of our role or level.
 
->Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+_Behaviours_:
+ - We ensure to treat everyone we encounter with respect independently of their skillsets and levels of ability
+ - Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+ - We respect the opinion of others and are gracious in our interactions, whether meeting physically or digitally, whether friend or stranger, whether employee, customer or commenter.
```

Comment:
As you know `uninclusive opinions` could be a divisive topic because outside of what is covered by law `uninclusive` is subject to interpretation based on culture, time and everyone's background. The Guardian is an independent newspaper and media organisation and as such may choose to publish articles or not moderate comments that may be felt by some as `uninclusive`. As an employee one have the right to (strongly) disagree with some editorial choice, a moderation decision, a customer review, or with an other employee's opinion but that disagreement should be expressed in a way which is respectful to the other party.